index.html

```diff
@@ -64,63 +70,85 @@ <h1 class="item1 h1">I’m Kirubel. Glad to see you!</h1>
       </div>
       <button>See Project</button>
     </ul>
-    <img src="./images/Snapshoot.png" alt="snapshot of website">
-    <p class="oldo">Multi-Post<br>Stories</p>
+    </div>
+  </div>
+  <div class="back">
+    <div class="bodydiv bodydiv2">
+    <p class="oldo">Multi-Post Stories</p>
     <ul>
       <div class="work1-li1">
-        <li class="work1-item1 work1-item3">CANOPAY</li>
-        <li class="work1-item1 work1-item2">Back End Dev</li>
+        <li class="work1-item1 work1-item3">FACEBOOK</li>
+        <li class="work1-item1 work1-item2">Full Stack End Dev</li>
         <li class="work1-item1 work1-item2">2015</li>
       </div>
       </ul>
-       <p class="pp">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+       <p class="pp">Experimental content creation feature that allows users to add 
+        to an existing story over the course of a day without spamming their friends.</p>
        <ul>
        <div class="work1-link">
       <li >HTML</li>
+      <li class="newlin">Ruby on rails</li>
       <li class="newlin">CSS</li>
       <li class="newlin">JavaScript</li>
       </div>
```

Comment:
:orange_circle: [OPTIONAL] Please make sure you use the proper background color on  each list item to match the Figma Design

Your Design
![properbackgroundstudent](https://user-images.githubusercontent.com/26736582/229287649-75674e1d-2ea3-4285-b1d7-8400fa9d9015.png)

Figma Design
![properbackground](https://user-images.githubusercontent.com/26736582/229287655-1eb773fe-f80d-46cb-8e64-5faa6e7edfe9.png)